src/Notifications/HubHelpers.cs

```diff
@@ -231,6 +231,17 @@ await _hubContext.Clients.Group(NotificationsHub.GetOrganizationGroup(
                 await _hubContext.Clients.User(pendingTasksData.Payload.UserId.ToString())
                     .SendAsync(_receiveMessageMethod, pendingTasksData, cancellationToken);
                 break;
+            case PushType.PolicyChanged:
+                var policyData = JsonSerializer.Deserialize<PushNotificationData<SyncPolicyPushNotification>>(notificationJson, _deserializerOptions);
```

Comment:
It’s understandable to follow the existing pattern, but I think it would be easier to maintain if we abstracted this into a private method.

```C#
                var policyData = JsonSerializer.Deserialize<PushNotificationData<SyncPolicyPushNotification>>(notificationJson, _deserializerOptions);

                if (policyData is null)
                {
                    break;
                }

                await _hubContext.Clients
                    .Group(NotificationsHub.GetOrganizationGroup(policyData.Payload.OrganizationId))
                    .SendAsync(_receiveMessageMethod, policyData, cancellationToken);
```